Tests/SentryProfilerTests/SentryProfilerSwiftTests.swift

```diff
@@ -92,60 +97,48 @@ class SentryProfilerSwiftTests: XCTestCase {
 
             // gather mock cpu usages and memory footprints
             for _ in 0..<mockUsageReadingsPerBatch {
-                self.timerWrapper.fire()
+                self.metricTimerWrapper.fire()
             }
 
     #if !os(macOS)
-            var shouldRecordFrameRateExpectation = false
-            var previousSystemTime = currentDateProvider.systemTime()
```

Comment:
We don't have to track this now that we are changing from recording frame info with the "current" timestamp instead of the "previous" one.